### ISSUE TITLE
I downloaded your code and ran the code according to the default hyperparameters, but found that the loss did not decrease. Do I need to do some other operations before running this code?

### ISSUE THREAD

**Issue by BeiMingYy** [Paranioar/SGRAF#13]
I downloaded your code and ran the code according to the default hyperparameters, but found that the loss did not decrease. Do I need to do some other operations before running this code?

**Comment by Paranioar** [comment]
> I downloaded your code and ran the code according to the default hyperparameters, but found that the loss did not decrease. Do I need to do some other operations before running this code?

Did you finish all the training process？ 
Firstly, please configure the environment according to our declarations in different code versions.
The loss of the SAF module declines at about 7-8 epoch, while for the SGR module it is at about 12-13 epoch.
More details can be found in [[About the loss]](https://github.com/Paranioar/SGRAF/issues/2).
